packages/suite/src/views/guide/GuideArticle.tsx

```diff
@@ -69,7 +69,7 @@ const GuideArticle = () => {
     useEffect(() => {
         const loadMarkdown = async () => {
             const file = await import(
-                `@trezor/suite-data/files/guide/en/trezor-suite/${article}.md`
+                `@trezor/suite-data/files/guide-temp/en/trezor-suite/${article}.md`
```

Comment:
Nope. I tried to explain this in the commit messsage. The `temp` is there because the whole directory will be deleted once we (you, most probably) edit the UI to use the articles fetched from GitBook.

In other words: we ultimately want the articles to be in `files/guide`. This is where Tomas Listiak commited the hardcoded articles. This PR wants to use the same directory, but because this time the content is auto-generated we need to git-ignore it. If we do that the current version of Guide would break as the articles would be gone.

There are two solutions:
- Move the hardocded articles aside temporarily to free `files/guide` directory to be populated by my scripts.
- Connect the UI to the auto-generated articles within this PR.

I chose the former. That is, I moved the hardcoded articles to `files/guide-temp`. Once you'll start using articles in `files/guide` we can safely delete `files/guide-temp`.

Not sure if it all makes sense. ./